### DISCUSSION TITLE
ebusd would not found my Vaillant SensoCOMFORT VRC 720 / VR 720

### DISCUSSION THREAD

**Discussion by pinkeswasser** [john30/ebusd#999] [Q&A]
Hello there,

i have a problem getting my ebusd running well. I am owner of a Vaillant heatpumpt, with the VRC 720 Controller.

It seems that ebusd find the heatpump and one other device, but not the VRC 720. I tried out with an other 720/2, but still the same problem.

My config:
`EBUSD_OPTS="--device=x.x.x.x:5000 --scanconfig=full --configpath=http://ebusd.eu/config/ --accesslevel=* --loglevel=debug --port 8888 --latency=20000 --httpport=8080 --htmlpath=/var/ebusd/html"`

The info:
`ebusctl i
version: ebusd 23.2.23.2
update check: OK
device: x.x.x.x:5000
access: *
signal: acquired
symbol rate: 45
max symbol rate: 111
min arbitration micros: 6
max arbitration micros: 234
min symbol latency: 4
max symbol latency: 4
scan: finished
reconnects: 0
masters: 3
messages: 2
conditional: 0
poll: 0
update: 0
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0517;HW=5103"
address 31: master #8, ebusd
address 36: slave #8, ebusd, scanning
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZ00;SW=0522;HW=5103"`

I read a lot, but cant't find a solution for it. I realised, that other users have an address 15 listed in the info, which describes the contoller. But not in my case, there is no address 15.

Anyone has an idea and could help me?

Thank you very much.

Best Regards,
Yannick
[ebusd.log](https://github.com/john30/ebusd/files/12480322/ebusd.log)

**Comment by iszczyg** [comment]
Hi, Try this:
https://github.com/john30/ebusd/issues/487#issuecomment-1001003007
I had the same problem and it solved my problem. 
Regards
Irek

**Reply by pinkeswasser** [reply]
Hi Irek,

thanks you very much, but thats not working for me.

The device is still not displayed in the list, so there is nothing to link.

It seems that in your case the device is found, but missing the correct config. Therefor it works.

I hope some else has a similar idea an know an solotion for it.

Best Regards,
Yannick

**Comment by VivLloyd** [comment]
What does **ebusctl info** return? This will list the addresses of all of the devices found on the bus and should help identify the missing message file e.g.
...
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0607;HW=5103", loaded "vaillant/08.hmu.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=BASV2;SW=0562;HW=6304", loaded "vaillant/15.basv.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZIO;SW=0202;HW=0103", loaded "vaillant/76.vwzio.csv"
'''
